apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Precondition.cs

```diff
@@ -20,7 +20,7 @@ namespace Google.Cloud.Firestore
     /// <summary>
     /// Immutable class representing a precondition for an update operation.
     /// </summary>
-    public sealed class Precondition : IEquatable<Precondition>
+    public sealed class Precondition
```

Comment:
This isn't strictly an Options class. Just checking you definitely meant to remove Equality from here?

Reply:
Yup, definitely.